apps/epic-web/src/inngest/functions/sanity/product/index.ts

```diff
@@ -43,6 +44,7 @@ export const BaseSanityProductSchema = z.object({
   title: z.string(),
   slug: z.string(),
   state: z.enum(['draft', 'active', 'unavailable']).default('draft'),
+  type: z.enum(['self-paced', 'live']).default('self-paced'),
```

Comment:
this zod schema enum has `live`, but the test mock has `live-event` -- which value should it be? Might be worth having a type or constant that these can import for consistency.